.github/workflows/android-build-verify.yml

```diff
@@ -1,5 +1,8 @@
 name: ID PASS - MOSIP Resident Application
 
+env:
+  backendServiceDefaultUrl: https://api.qa-121.mosip.net
```

Comment:
can we make the URL as mentioned below to make the same generic.
https://api.sandbox.mosip.net

Reply:
Changes are made